### PR TITLE
Add middle boxes using Q bit signal for reordering fixup as a concern.

### PR DESCRIPTION
**Pull request by igorlord**
Adding this as an ossification concern for now.  Happy to find a different place it, if needed.

### CODE REVIEW

File: draft-ferrieuxhamchaoui-quic-lossbits.md [hunk 1]
```diff
@@ -352,6 +352,11 @@ greasing in {{QUIC-TRANSPORT}}.  Namely, implementations MUST NOT include
 loss_bits transport parameter for a random selection of at least one in every 16
 connections.
 
+It is possible to observe packet reordering near the edge of the square signal.
+A middle box might observe the signal and try to fix packet reordering that it
+can identify.  Only a small fraction of reordering can be fixed using this
```
Comment (dtikhonov) [inline draft-ferrieuxhamchaoui-quic-lossbits.md]: I'd split the last sentence into two.

**Review by dtikhonov — APPROVED** [review]

**Review by ferrieux — COMMENTED** [review]
In addition to the spin bit, there's also the IP.id in IPv4.

File: draft-ferrieuxhamchaoui-quic-lossbits.md (hunk 1 above)
Comment (igorlord) [reply draft-ferrieuxhamchaoui-quic-lossbits.md]: Split it into two and recombined the first part with the previous sentence.

**Comment by igorlord** [comment]
@ferrieux 
> In addition to the spin bit, there's also the IP.id in IPv4.

If IP.id often helpful for reordering fixup?  Linux sets IP.id=0, unless there are fragments.

**Pull request merged**